app panics on reference to a map which has <string, nil> item.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1545b8f]

goroutine 1 [running]:
github.com/jhump/protoreflect/dynamic.(*Message).GetMessageDescriptor(...)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/dynamic_message.go:167
github.com/jhump/protoreflect/dynamic.validElementFieldValueForRv(0xc0002df110, 0x1855420, 0x0, 0x16, 0x171b160, 0xc0000a84d0, 0x0, 0x0)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/dynamic_message.go:1936 +0xb5f
github.com/jhump/protoreflect/dynamic.validElementFieldValue(0xc0002df110, 0x1855420, 0x0, 0x171b160, 0xc0000a84d0, 0x0, 0x0)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/dynamic_message.go:1883 +0xaa
github.com/jhump/protoreflect/dynamic.(*Message).putMapField(0xc0000ab530, 0xc0002df030, 0x171b160, 0xc0000a84c0, 0x1855420, 0x0, 0x0, 0xc0000d90d8)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/dynamic_message.go:1179 +0x105
github.com/jhump/protoreflect/dynamic.(*Message).addRepeatedField(0xc0000ab530, 0xc0002df030, 0x1855420, 0xc000168d20, 0x0, 0x0)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/dynamic_message.go:1591 +0x3ab
github.com/jhump/protoreflect/dynamic.mergeField(0xc0000ab530, 0xc0002df030, 0x1855420, 0xc000168d20, 0x0, 0x1855420)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/merge.go:93 +0x66d
github.com/jhump/protoreflect/dynamic.(*Message).unmarshalKnownField(0xc0000ab530, 0xc0002df030, 0xc0002df002, 0xc0000d92c8, 0x0, 0x0)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/binary.go:609 +0xea
github.com/jhump/protoreflect/dynamic.(*Message).unmarshal(0xc0000ab530, 0xc0003cd2c8, 0x1008500, 0xc00001e000, 0x17713e0)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/binary.go:414 +0xa9
github.com/jhump/protoreflect/dynamic.(*Message).UnmarshalMerge(0xc0000ab530, 0xc00003c5d0, 0x26, 0x26, 0x1, 0x25d7980)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/binary.go:390 +0x78
github.com/jhump/protoreflect/dynamic.(*Message).Unmarshal(0xc0000ab530, 0xc00003c5d0, 0x26, 0x26, 0xc0000ab530, 0x1)
        /go/pkg/mod/github.com/jhump/protoreflect@v1.1.0/dynamic/binary.go:379 +0x60
google.golang.org/grpc/encoding/proto.codec.Unmarshal(0xc00003c5d0, 0x26, 0x26, 0x1855420, 0xc0000ab530, 0xc0000d93f0, 0x10dda2c)
        /go/pkg/mod/google.golang.org/grpc@v1.20.1/encoding/proto/proto.go:88 +0x1fc
google.golang.org/grpc.recv(0xc0000b4800, 0x2651150, 0x1f27af0, 0xc0002ba200, 0x0, 0x0, 0x1855420, 0xc0000ab530, 0x400000, 0x0, ...)
        /go/pkg/mod/google.golang.org/grpc@v1.20.1/rpc_util.go:688 +0x10f
google.golang.org/grpc.(*csAttempt).recvMsg(0xc0000c0280, 0x1855420, 0xc0000ab530, 0x0, 0xc0000d9698, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.20.1/stream.go:870 +0xed
google.golang.org/grpc.(*clientStream).RecvMsg.func1(0xc0000c0280, 0x0, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.20.1/stream.go:721 +0x46
google.golang.org/grpc.(*clientStream).withRetry(0xc00012c120, 0xc0003cd6a0, 0xc0003cd690, 0xc0001b9a10, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.20.1/stream.go:577 +0x7c
google.golang.org/grpc.(*clientStream).RecvMsg(0xc00012c120, 0x1855420, 0xc0000ab530, 0x0, 0x0)
        /go/pkg/mod/google.golang.org/grpc@v1.20.1/stream.go:720 +0x100
google.golang.org/grpc.invoke(0x196e3c0, 0xc0000ab410, 0xc00003c5a0, 0x28, 0x1855420, 0xc0000ab440, 0x1855420, 0xc0000ab530, 0xc0002dc000, 0x0, ...)
        /go/pkg/mod/google.golang.org/grpc@v1.20.1/call.go:73 +0x141
google.golang.org/grpc.(*ClientConn).Invoke(0xc0002dc000, 0x196e3c0, 0xc0000ab410, 0xc00003c5a0, 0x28, 0x1855420, 0xc0000ab440, 0x1855420, 0xc0000ab530, 0x0, ...)
        /go/pkg/mod/google.golang.org/grpc@v1.20.1/call.go:37 +0x1b4
github.com/ktr0731/evans/grpc.(*client).Invoke(0xc0002f6040, 0x196e3c0, 0xc0000ab410, 0xc0002f8630, 0x27, 0x1855420, 0xc0000ab440, 0x1855420, 0xc0000ab530, 0x0, ...)
        /go/src/github.com/ktr0731/evans/grpc/grpc.go:176 +0x1a0
github.com/ktr0731/evans/usecase.(*dependencyManager).CallRPC(0xc0002ec660, 0x196e340, 0xc0000b6000, 0x195eae0, 0xc00000e010, 0xc0000c2480, 0xe, 0x0, 0x141f38c)
        /go/src/github.com/ktr0731/evans/usecase/call_rpc.go:248 +0xad4
github.com/ktr0731/evans/usecase.CallRPC(...)
        /go/src/github.com/ktr0731/evans/usecase/call_rpc.go:17
github.com/ktr0731/evans/repl.(*callCommand).Run(0x1f27af0, 0x195eae0, 0xc00000e010, 0xc0000b42d0, 0x1, 0x1, 0xc0000a8320, 0x1f27af0)
        /go/src/github.com/ktr0731/evans/repl/commands.go:210 +0x8c
github.com/ktr0731/evans/repl.(*REPL).runCommand(0xc0002dab80, 0xc0001b9198, 0x4, 0xc0000b42d0, 0x1, 0x1, 0x0, 0x0)
        /go/src/github.com/ktr0731/evans/repl/repl.go:148 +0x198
github.com/ktr0731/evans/repl.(*REPL).Run(0xc0002dab80, 0x196e300, 0xc0002dab40, 0x0, 0x0)
        /go/src/github.com/ktr0731/evans/repl/repl.go:110 +0x165
github.com/ktr0731/evans/mode.RunAsREPLMode(0xc000089f50, 0x196ec80, 0xc0002c3190, 0xc0002c6050, 0x0, 0x0)
        /go/src/github.com/ktr0731/evans/mode/repl.go:85 +0x669
github.com/ktr0731/evans/app.(*App).run(0xc0000b5ee0, 0xc0000a40d0, 0x5, 0x5, 0x0, 0x0)
        /go/src/github.com/ktr0731/evans/app/app.go:123 +0x453
github.com/ktr0731/evans/app.(*App).Run(0xc0000b5ee0, 0xc0000a40d0, 0x5, 0x5, 0xc0000b5ec0)
```

--- a/go.mod
+++ b/go.mod
@@ -14,7 +14,7 @@ require (
 	github.com/gorilla/websocket v1.4.1 // indirect
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/hashicorp/go-version v1.2.0
-	github.com/jhump/protoreflect v1.5.0
+	github.com/jhump/protoreflect v1.5.1-0.20191024213132-10815c273d3f
 	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
 	github.com/k0kubun/pp v3.0.1+incompatible
 	github.com/ktr0731/go-multierror v0.0.0-20171204182908-b7773ae21874

